website/server/controllers/api-v3/hall.js

```diff
@@ -11,17 +11,108 @@ import _ from 'lodash';
  * Contributors of tier 8 or higher can use this route.
  */
 
+/**
+ * @apiDefine NoAuthHeaders Missing authentication headers
```

Comment:
these errors can be moved somewhere more generic since they're not only used here?